department-of-veterans-affairs/va.gov-cms
# VACMS-7357: Make VAMC system service description required

- Updated "VAMC System Health Service" content type "VAMC system service description" (field_body) field and made it required. 
- Updated Drupal spec tool and behat.

## Description

closes #7357 

## Testing done

Passes all tests.

## Screenshots
VAMC System Health Service content type:
![Screenshot from 2022-01-05 18-38-17](https://user-images.githubusercontent.com/846871/148305108-12600c10-19e3-40b5-aac8-740053eb18d3.png)

VAMC System Health Service node:
![Screenshot from 2022-01-05 18-34-08](https://user-images.githubusercontent.com/846871/148304817-b4113463-5397-4032-895c-22b1c14b323d.png)


## QA steps

As user administrator
1. Go to `/admin/structure/types/manage/regional_health_care_service_des/fields/node.regional_health_care_service_des.field_body` to verify and confirm the "VAMC system service description" field required field checkbox is selected
2. Go to `/node/29068/edit` (or any VAMC System Health Service node) to verify and confirm the "VAMC system service description" field is required

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
